### DISCUSSION TITLE
Question on 5v power

### DISCUSSION THREAD

**Discussion by n9jcv** [TomNisbet/TommyPROM#41] [Q&A] [answered]
I am starting to build.  I have a laptop and the nano connected thru a short usb cable. 

I measure the voltage on the 5v pin and it is 4.355.  From the pics and schematic it seems the entire circuit is powered by the 5v pin on the arduino.  But I am concerened the voltage is too low.  

Can I connect a 5v supply to the breadboard or will that damage the laptop or cause ground loop issues?

Thanks for your help

**Comment by TomNisbet** [comment, marked as answer]
I always power it from the Arduino and have programmed dozens of chips just fine. The data sheet says 4.5 volts minimum, but I haven't ever had a problem.

Now I'm curious. I need to remember to measure the voltage the next time I have one hooked up to see what mine is reading.

**Reply by n9jcv** [reply]
Ok thanks I will continue to build and test.  I checked 3 computers and all give 4.3 to 4.4 so hopefully it will work fine.  I tried 3 different usb cables as well.  All short ones.  

Thank you for sharing all of this as well!!!
